Applications/Games/STAR WARS Battlefront II/Local/script.js

```diff
@@ -0,0 +1,22 @@
+include(["Engines", "Wine", "QuickScript", "LocalInstallerScript"]);
+include(["Engines", "Wine", "Engine", "Object"]);
+include(["Utils", "Functions", "Filesystem", "Files"]);
+
+new LocalInstallerScript()
+    .name("STAR WARS™ Battlefront™ II")                       
+    .editor("Pandemic Studiosr")                               
+    .author("ImperatorS79")                     
+    .category("Games")                           
+    .executable("LaunchBFII.exe")                  
+    .postInstall(function(wine, wizard) {
+        new Downloader()
+            .wizard(wizard)
+            .url("http://static.dolimg.com/mh_netstorage/lucasfilm/patches/pc/BFIIUpdateInt1_1.exe")
+            .checksum("60eaddfaba1bc71fe8bbbb560f8da229748cfaa8")
+            .to(wine.prefixDirectory + "drive_c/Program Files/LucasArts/Star Wars Battlefront II/BFIIUpdateInt1_1.exe")
+            .get();
+            
+        wine.runInsidePrefix("/Program Files/LucasArts/Star Wars Battlefront II/BFIIUpdateInt1_1.exe");
```

Comment:
Maybe you should add a short comment explaining that the downloaded and installed file is patch 1.1, I know that it should be clear from this PR, but if someone looks at the script later on, it won't be as clear then.